Dronecode-SDK-Swift/Info.swift

```diff
@@ -0,0 +1,95 @@
+//
```

Comment:
Can we remove the copyright? :sweat_smile:

Reply:
done.